_source/logzio_collections/_security-sources/crowdstrike.md

```diff
@@ -139,19 +141,14 @@ Upon deployment, the Crowdstrike connector connects to your Crowdstrike account
    </label>
    ```
 
-   <!-- info-box-start:info -->
-   The default configuration for the log level is set to **info**. The **log_level** setting defines which events are recorded in the log. In order of verbosity, the log level can be defined as **fatal**, **error**, **warn**, **info**, **debug** or **trace**. The **info** level records all events categorized as **info** and higher in verbosity. To learn more about the settings used in the configuration file, see [Logz.io plugin for Fluentd](https://github.com/logzio/fluent-plugin-logzio).
-   {:.info-box.note}
-   <!-- info-box-end -->
-
-   <!-- info-box-start:info -->
+   The default configuration for the log level is set to **info**. The **log_level** setting defines which events are recorded in the log. In order    of verbosity, the log level can be defined as **fatal**, **error**, **warn**, **info**, **debug** or **trace**. The **info** level records all    events categorized as **info** and higher in verbosity. To learn more about the settings used in the configuration file, see [Logz.io plugin for    Fluentd](https://github.com/logzio/fluent-plugin-logzio).
+   
    If you change `type` in the `@type record_transformer` section, the data is not parsed into the relevant fields for the Crowdstrike integration.
-   {:.info-box.note}
-   <!-- info-box-end -->
-
-4. Replace ```<<LOGZIO_LISTENER>>``` with the Listener URL for your account.
-5. Replace ```<LOGZIO_SHIPPING_TOKEN>``` with your token value.
-6. Save the changes.
+   
+   {% include /log-shipping/listener-var.html %}
```

Comment:
the 2 includes should be numbered steps

Reply:
I removed the numbering intentionally, as these steps would otherwise appear as empty numbers in the portal. Same for the bullet points, it shows empty bullet points in-app.